sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -129,7 +129,6 @@ static void handleVisitedNodes(std::vector<Command *> &Visited) {
   for (Command *Cmd : Visited) {
     if (Cmd->MMarks.MToBeDeleted) {
       Cmd->getEvent()->setCommand(nullptr);
-      Cmd->getEvent()->cleanupDependencyEvents();
```

Comment:
It looks like you removed all uses of `cleanupDependencyEvents`. Can we remove the definition of this function?

Reply:
This function is used in Command::~Command()

Reply:
https://github.com/intel/llvm/blob/sycl/sycl/source/detail/scheduler/commands.hpp#L190
I can't find it. Can you point to the exact location in the source code?

Reply:
This PR changes ~Command and this function is called there
https://github.com/intel/llvm/pull/5401/files#diff-5e76d6e04c052ad4472e6884f7f9756e9696fe1f02ae959ed31a327457c33f9aL190